save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AboutUsView.kt

```diff
@@ -0,0 +1,199 @@
+                                    jso {
+                                        this.children = saveourtoolDescription
+                                        this.className = "flex-wrap"
+                                    }
+                                )
+                            )
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    @Suppress("MAGIC_NUMBER")
+    private fun ChildrenBuilder.renderDevelopers() {
+        div {
+            h4 {
+                className = ClassName("text-center mb-2 mt-3")
+                +"core team"
+            }
+            div {
+                className = ClassName("mt-3 d-flex justify-content-around align-items-center")
+                div {
+                    for (rowIndex in 0..2) {
+                        div {
+                            className = ClassName("row")
+                            for (colIndex in 0..2) {
+                                div {
+                                    className = ClassName("col-4 p-2")
+                                    developers.getOrNull(3 * rowIndex + colIndex)?.let {
+                                        renderDeveloperCard(it)
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    private fun ChildrenBuilder.renderDeveloperCard(developer: Developer) {
+        devCard {
+            div {
+                className = ClassName("p-3")
+                div {
+                    className = ClassName("d-flex justify-content-center")
+                    img {
+                        src = "$GITHUB_AVATAR_LINK${developer.githubNickname}?size=$DEFAULT_AVATAR_SIZE"
+                        className = ClassName("img-fluid border border-dark rounded-circle m-0")
+                    }
+                }
+                div {
+                    className = ClassName("mt-2")
+                    h5 {
+                        className = ClassName("d-flex justify-content-center")
+                        +developer.name
+                    }
+                    h6 {
+                        className = ClassName("text-center")
+                        +developer.description
+                    }
+                    a {
+                        className = ClassName("d-flex justify-content-center")
+                        href = "$GITHUB_LINK${developer.githubNickname}"
+                        +developer.githubNickname.take(MAX_NICKNAME_LENGTH)
+                    }
+                }
+            }
+        }
+    }
+
+    companion object :
+        RStatics<AboutUsViewProps, AboutUsViewState, AboutUsView, Context<RequestStatusContext>>(AboutUsView::class) {
+        private const val DEFAULT_AVATAR_SIZE = "175"
+        private const val GITHUB_AVATAR_LINK = "https://avatars.githubusercontent.com/"
```

Comment:
Maybe take our avatars from the Save, and not GitHub?

Reply:
Not everyone has his face on saveourtool, but everybody has it on GitHub

Reply:
GitHub, definitely